src/main/scala/is/hail/methods/PCRelate.scala

```diff
@@ -255,7 +255,7 @@ class PCRelate(maf: Double, blockSize: Int) extends Serializable {
 
     val stddev = variance.sqrt()
 
-    (gram(centeredG) /:/ gram(stddev)) / 4.0
+    (gram(centeredG) / gram(stddev)) / 4.0
```

Comment:
Seems I'm not up on the Breeze syntax.  What's this one about?

Reply:
These are BlockMatrices. I changed the BlockMatrix ops above to be consistent between Python and Scala, so `/` is element-wise division whether by a BlockMatrix, Array, or Double.